l10n_ar_withholding_ux/models/account_tax.py

```diff
@@ -80,6 +80,7 @@ class AccountTax(models.Model):
         'tax_withholding_id',
         'Rules',
     )
+    ratio = fields.Float(required=True, default=100, help="Ratio to apply to tax base amount")
```

Comment:
remove default and required, extend help
```suggestion
    ratio = fields.Float(help="Ratio to apply to tax base amount. If left empty will be 100 % ratio")
```